<?xml version="1.0"?>
<launch>
    <arg name="file_name" default="$(find route_maker)/config/wall0529.yaml"/>
    <!-- <arg name="map" default="$(find route_maker)/map/dkan.yaml"/> -->
    <arg name="map" default="$(find route_maker)/map/Urange20Xmin25Ymin25.yaml"/>

    <node pkg="route_maker" type="wall_recoder.py" name="area_recoder" output="screen">
        <param name="FILE_NAME" value="$(arg file_name)"/>
    </node>

    <node pkg="map_server" type="map_server" name="map_server" args="$(arg map)"/>
</launch>
